src/app/modules/admin-page/admin-page.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from '@angular/core';
+import { FirebaseService } from 'app/core/services/firebase.service';
+import { Dish } from 'app/core/interfaces/dish';
+
+@Component({
+  selector: 'app-admin-page',
+  templateUrl: './admin-page.component.html',
+  styleUrls: ['./admin-page.component.scss']
+})
+export class AdminPageComponent implements OnInit {
+  public image;
+  public dish: Dish = {
+    name: '',
+    type: '',
+    todaymenu: false,
+    price: 0,
+    weight: '',
+    img: '',
+    info: ''
+  };
+
+  public filterProp = [
```

Comment:
You have this configuration in few files. So, to avoid code duplication, put it to a separate file like config.ts and import it where you need. And create interface for it.